tests/integration/backward_compatible/proxy/map_test.py

```diff
@@ -596,6 +617,31 @@ def test_values_with_predicate(self):
     def test_str(self):
         self.assertTrue(str(self.map).startswith("Map"))
 
+    def test_add_interceptor(self):
+        interceptor = MapGetInterceptor("hell")
```

Comment:
I don't know why 666 was used as the factory ID in the test jar, but we probably can rename this better :D

Reply:
How about `hello`?

Reply:
Yeah, probably :smile: